web/src/stream_settings_ui.js

```diff
@@ -234,20 +236,38 @@ export function add_sub_to_table(sub) {
         // ID isn't known yet.  These are appended to the top of the
         // list, so they are more visible.
         stream_ui_updates.row_for_stream_id(sub.stream_id).trigger("click");
+        const stream_url = hash_util.by_stream_url(sub.stream_id);
         const context = {
             banner_type: compose_banner.SUCCESS,
             classname: "stream_creation_confirmation",
             stream_name: sub.name,
-            stream_url: hash_util.by_stream_url(sub.stream_id),
+            stream_url,
         };
         $("#stream_settings .stream-creation-confirmation-banner").html(
             render_stream_creation_confirmation_banner(context),
         );
         stream_create.reset_created_stream();
+        // goto topic `stream events` of the newly created stream
+        window.location.href = stream_url + "/topic/stream.20events";
```

Comment:
You can use `hash_util.by_stream_topic_url` here.

Reply:
Done